devstudio/installer/category.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+	<description>
+		JBoss Developer Studio Integration Stack Installer
+	</description>
+
+	<!-- copy IU's from JBDS Core Installer -->
+	<feature id="org.jboss.tools.bpel.feature" />
+	<feature id="org.testng.eclipse" />
+	<bundle id="com.jboss.devstudio.core.package" />
```

Comment:
yes it would contains full copy of JBDS package + IS IUs, but then we just replace original jbdevsudio p2 repo inside installerjar!/jbds folder with this one. It is the simplest way to have all additional dependencies like zest to be added toIS installer.